src/main/kotlin/com/example/demo/controller/PostsController.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.demo.controller
+
+import com.example.demo.domain.Posts
+import com.example.demo.service.PostsService
+import org.springframework.stereotype.Controller
+import org.springframework.ui.Model
+import org.springframework.web.bind.annotation.*
+
+@Controller
+@RequestMapping("/home")
+class PostsController(private val postsService: PostsService) {
+    @GetMapping
+    fun index(model: Model): String {
+        model.addAttribute("posts", postsService.findAll())
+        println(postsService.findAll())
+        postsService.findAll().forEach{i->
```

Comment:
ktlintで縛っていないのでアレですが、IntellIJのreformat機能くらいは使っても良さそうですね
多分こうなります

```suggestion
        postsService.findAll().forEach{ i ->
```

Reply:
forEachなので、i(index)というよりはpostですね
(デバッグ用に適当に付けただけだとは思いますが)

```suggestion
        postsService.findAll().forEach{ post ->
```